src/main/java/org/elasticsearch/discovery/srv/SrvUnicastHostsProvider.java

```diff
@@ -163,15 +162,15 @@ protected Resolver buildResolver(Settings settings) {
     }
 
     protected List<DiscoveryNode> lookupNodes() throws TextParseException {
-        List<DiscoveryNode> discoNodes = Lists.newArrayList();
+	List<DiscoveryNode> discoNodes = new ArrayList<DiscoveryNode>();
```

Comment:
Out of curiosity, is `new ArrayList<DiscoveryNode>()` generally preferable to `Lists.newArrayList()`?